notifiers/slackNotifier.go

```diff
@@ -0,0 +1,95 @@
+package notifiers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/multiversx/mx-chain-keys-monitor-go/core"
+	httpSDK "github.com/multiversx/mx-sdk-go/core/http"
+)
+
+const (
+	slackBoldFormat       = "*%s*"
+	slackBoldedLinkFormat = "*<%s|%s>*"
+)
+
+type slackRequest struct {
+	Text string `json:"text"`
+}
+
+type slackNotifier struct {
+	httpClientWrapper HTTPClientWrapper
+	secret            string
+}
+
+// NewSlackNotifier will create a new Slack notifier
+func NewSlackNotifier(url string, secret string) *slackNotifier {
+	return &slackNotifier{
+		httpClientWrapper: httpSDK.NewHttpClientWrapper(nil, url),
```

Comment:
not related to this PR, we might need to have another constructor in http sdk with default client or something, instead of passing a nil var

Reply:
Agree with @ssd04 about this :+1: